src/libs/ReportUtils.ts

```diff
@@ -2070,7 +2075,7 @@ function getModifiedExpenseOriginalMessage(oldTransaction: OnyxEntry<Transaction
 /**
  * Returns the parentReport if the given report is a thread.
  */
-function getParentReport(report: OnyxEntry<Report>): OnyxEntry<Report> | EmptyObject {
+function getParentReport(report: OnyxEntry<Report> | EmptyObject): OnyxEntry<Report> | EmptyObject {
```

Comment:
```suggestion
function getParentReport(report: OnyxEntry<Report>): OnyxEntry<Report> | EmptyObject {
```

Is possible to avoid doing ` | EmptyObject`?